server/src/packages/pas-form/pas-form.service.ts

```diff
@@ -17,11 +17,13 @@ export class PasFormService {
         dcp_dcp_applicantinformation_dcp_pasform,
         dcp_dcp_applicantrepinformation_dcp_pasform,
         dcp_dcp_projectbbl_dcp_pasform($filter=statecode eq 0),
-        dcp_dcp_projectaddress_dcp_pasform
+        dcp_dcp_projectaddress_dcp_pasform,
+        dcp_projectname($select=dcp_projectname)
     `);
 
-    // CRM feature for providing the human-readable value for the related resource
-    const { '_dcp_projectname_value@OData.Community.Display.V1.FormattedValue': dcp_projectname } = pasForm;
+    // dcp_projectname is the name of the CRM navigation link.
```

Comment:
did you mean to type another property here?

Reply:
Oh, I see. So pasForm.dcpProjectname is an object with a property of the same name?
```
pasForm = {
    dcp_projectname: {
       dcp_projectname: "My project name", 
    }
}
```

And is the nested `dcp_projectname value the longform value, like "My project name", whereas `_dcp_projectname_value@OData.Community.Display.V1.FormattedValue` is the project id, like `20202MK1`?

Reply:
Yes exactly. It's a little weird but that's how it's set up in CRM.